package/styles/DataTableRowMenu.css

```diff
@@ -0,0 +1,6 @@
+root: {
+    position: fixed,
+    border: 1px solid light-dark(var(--mantine-color-gray-3), var(--mantine-color-dark-4)),
+    overflow: hidden,
+    transition: all .15s ease,
```

Comment:
Also some cases like this which need to be converted to css syntax `.root {` and not `root: {`